elastic/kibana
# [Discover] Update formatter for aggregate_metric_double field values

## Summary

Closes #134224

This PR updates formatter for `aggregate_metric_double` field values.

### How to test locally
- Check out https://github.com/elastic/elasticsearch/tree/kibana-downsampling-branch next to your kibana working copy
- Start ES with yarn es source instead of snapshot (this will build the given branch)
- Clone https://github.com/csoulios/downsampling-demo
- Run the commands in [downsample-index.es](http://downsample-index.es/) (indexing the actual data is too much for dev console, so you have to do this via curl curl -XPOST http://elastic:changeme@localhost:9200/sample-01/_bulk\?pretty -H 'Content-Type: application/json' --data-binary @sample-01-raw.json)
- After running all commands, there should be sample-01-rollup which has the rolled up data (for the data view make sure to explicitly list all indices to include the rollup index - sample-01,sample-01-rollup

- Expand the document and see formatted result

![DEF17EAE-95B4-4236-89EF-63F607AE242D_4_5005_c](https://user-images.githubusercontent.com/39378793/185660492-52d72c80-9e75-4bfe-a8c3-b5a0e18e40bf.jpeg)



- Before this changes



![919B6C5E-FFAB-43E9-9ED7-A196B3635E33_4_5005_c](https://user-images.githubusercontent.com/39378793/185661465-86742d00-67c1-4289-bc99-9d6568b028ed.jpeg)


### Checklist
